test/onnx/gen_onnx.py

```diff
@@ -6154,44 +6153,34 @@ def scatter_add_test():
     return ([node], [x, i, u], [y])
 
 
+@onnx_test()
+def scatter_add_test():
+    return make_scatter_elements_test("add")
+
+
 @onnx_test()
 def scatter_mul_test():
-    x = helper.make_tensor_value_info('data', TensorProto.FLOAT, [3, 4, 5, 6])
-    i = helper.make_tensor_value_info('indices', TensorProto.INT32,
-                                      [2, 3, 4, 5])
-    u = helper.make_tensor_value_info('update', TensorProto.FLOAT,
-                                      [2, 3, 4, 5])
-    y = helper.make_tensor_value_info('y', TensorProto.FLOAT, [3, 4, 5, 6])
+    return make_scatter_elements_test("mul")
```

Comment:
The corresponding onnx files to these five tests need to be created and added to the commit.

Reply:
ONNX files for none, add, and mul reductions existed before this PR, which adds tests for min and max.